src/libs/notifications.js

```diff
@@ -65,21 +65,16 @@ const NotificationDisplay = ({ id }) => {
   const onLast = notificationNumber + 1 === notifications.length;
 
   const { title, message, detail, type, action, onDismiss } = notifications[notificationNumber];
-  const [baseColor, ariaLabel] = Utils.switchCase(
+  const [baseColor, ariaLabel] = switchCase(
     type,
     ['success', () => [colors.success, 'success notification']],
     ['info', () => [colors.accent, 'info notification']],
     ['welcome', () => [colors.accent, 'welcome notification']],
     ['warn', () => [colors.warning, 'warning notification']],
     ['error', () => [colors.danger, 'error notification']],
-    [Utils.DEFAULT, () => [colors.accent, 'notification']]
-  );
-  const iconType = Utils.switchCase(
-    type,
-    ['success', () => 'success-standard'],
-    ['warn', () => 'warning-standard'],
-    ['error', () => 'error-standard']
+    [DEFAULT, () => [colors.accent, 'notification']]
   );
+  const iconType = switchCase(type, ['success', () => 'success-standard'], ['warn', () => 'warning-standard'], ['error', () => 'error-standard']);
```

Comment:
... and here.

Reply:
I don't see a default case here on dev:
https://github.com/DataBiosphere/terra-ui/blob/dev/src/libs/notifications.js#L77

Reply:
👍 The way the diff ended up after the formatting confused me.